src/Umbraco.Web.UI/Views/Partials/Grid/Bootstrap2-Fluid.cshtml

```diff
@@ -64,21 +64,32 @@
         JObject cfg = contentItem.config;
 
         if(cfg != null)
-            foreach (JProperty property in cfg.Properties()) {
-                attrs.Add(property.Name + "='" + property.Value.ToString() + "'");
+            foreach (JProperty property in cfg.Properties())
+            {
+                var propertyValue = TemplateUtilities.CleanForXss(property.Value.ToString());
+                if (string.IsNullOrWhiteSpace(propertyValue) == false)
```

Comment:
You're right about the properties .. I guess you would expect attributes even without values to be added on the lement. 
I'll keep it in for the css class though, as that would give you invalid html no matter what.